src/azuredeployP0.json

```
@@ -15,16 +15,19 @@
     "functionAppName": {
       "type": "string"
     },
-    "functionAppRuntimeVersion": {
+    "runtimeName": {
       "type": "string",
       "allowedValues": [
-        "DOTNET-ISOLATED|6.0",
-        "Python|3.10",
-        "Java|17",
-        "Node|18",
-        "PowerShell|7.2"
+        "DotNet-Isolated",
```

nitpick: I think the recommended spelling of .NET is "Dotnet" not "DotNet".
```suggestion
        "Dotnet-Isolated",
```

---

@davidmrdavid -- Should we use `Dotnet-isolated`? Or `Dotnet-Isolated`? I am inclining for the first option.

---

I would actually make this all lowercase. I think that'd be a simpler rule to follow, no?
Overall, no blockers on my end though. I'm 'ok' with `Dotnet-isolated` and `Dotnet-Isolated` too and I won't argue against either.

---

I will keep is as `Dotnet-isolated` to be consistent with other language offerings like `Python` and `Java`.